src/utils/metrics-calculator.ts

```
@@ -9,163 +9,103 @@ export class MetricsCalculator {
-          addComplexity = 1 + nestingLevel;
+          cyclomaticComplexity++;
+          addCognitiveComplexity = 1 + nestingLevel;
           break;
-          
+
+        case SyntaxKind.TryStatement:
+          newNestingLevel++;
+          break;
+
         case SyntaxKind.BinaryExpression:
           const binaryExpr = currentNode as any;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)